### PR TITLE
Update JetBrains plugin page

### PR DESCRIPTION
**Pull request by parlough**
- Formats and restructures some of the content to utilize our current utilities.
- Updates some moved and/or outdated links
- Update IntelliJ setup instructions for new(er) welcome screen
- Add a mention of `dart create` in relation to templates
- Switch from PhpStorm help to WebStorm help
- Include a link to the plugin earlier on

### CODE REVIEW

File: src/tools/jetbrains-plugin.md [hunk 1]
```diff
@@ -14,12 +14,11 @@ Whichever JetBrains IDE you choose for Dart development,
...
 and find more information when you need it.
 
-<aside class="alert alert-info" markdown="1">
-  **Note:**
+{{site.alert.note}}
   [WebStorm,](https://www.jetbrains.com/webstorm/)
   a JetBrains IDE for client-side development,
-  comes with the Dart plugin pre-installed.
-</aside>
+  comes with the Dart plugin pre-installed.
```
Comment (kwalrath) [inline src/tools/jetbrains-plugin.md:20]:
```suggestion
  comes with the Dart plugin pre-installed. 
```

File: src/tools/jetbrains-plugin.md [hunk 2]
```diff
@@ -31,19 +30,20 @@ Then install the Dart plugin and tell it where to find the Dart SDK.
 
 Install a JetBrains IDE if you don't already have one.
```
Comment (kwalrath) [inline src/tools/jetbrains-plugin.md:31]: I think we need to be clearer that this is an OR choice. Maybe:

```suggestion
Install a JetBrains IDE if you don't already have one. Choose one:
```

File: src/tools/jetbrains-plugin.md [hunk 3]
```diff
@@ -31,19 +30,20 @@ Then install the Dart plugin and tell it where to find the Dart SDK.
...
-* <a href="https://www.jetbrains.com/idea/download/"
-  target="_blank" rel="noopener">Download IntelliJ IDEA</a> or,
-  to try out the latest Dart language features,
-  [install IntelliJ IDEA EAP.](https://confluence.jetbrains.com/display/IDEADEV/EAP)
-* Or <a href="https://www.jetbrains.com/products.html"
-  target="_blank" rel="noopener">find another JetBrains product.</a>
+* [Download IntelliJ IDEA.][IDEA]{:target="_blank" rel="noopener"}
+* To try out the latest Dart language features and IntelliJ functionality,
+  [install IntelliJ IDEA EAP.][IDEA EAP]{:target="_blank" rel="noopener"}
+* [Find another Jetbrains product.][Other]{:target="_blank" rel="noopener"}
```
Comment (kwalrath) [inline src/tools/jetbrains-plugin.md:36]:
```suggestion
* [IntelliJ IDEA][IDEA]{:target="_blank" rel="noopener"}
* [IntelliJ IDEA EAP][IDEA EAP]{:target="_blank" rel="noopener"}
  (for early access to the latest Dart language features and IntelliJ functionality)
* [Another JetBrains product][Other]{:target="_blank" rel="noopener"}
```

**Review by kwalrath — APPROVED** [review]
I'd like a couple of minor fixes, but other than that, LGTM. Thank you, @parlough!

File: src/tools/jetbrains-plugin.md (hunk 3 above)
Comment (parlough) [reply src/tools/jetbrains-plugin.md:36]: I wasn't a fan of the periods, so thanks for this!

Comment (kwalrath) [reply src/tools/jetbrains-plugin.md]: > I wasn't a fan of the periods, so thanks for this!

:) Unlike some other companies, we use periods for list items that are complete sentences (but not generally for other list items). I think we used to use periods for more kinds of list items, but that's just silly. https://developers.google.com/style/lists#capitalization-and-end-punctuation

**Pull request merged**